### ISSUE TITLE
K8SSAND-631 ⁃ Make the Reaper image configurable

### ISSUE THREAD

**Issue by jsanda** [k8ssandra/k8ssandra-operator#24]
This an extension of #23 but focuses on the Reaper CRD. I am breaking this out into a separate ticket because the work can happen separately and possibly in parallel.

┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-631) by [Unito](https://www.unito.io)
┆epic: Reaper Controller Integration
┆fixVersions: k8ssandra-operator-v1.0.0-alpha.3
┆friendlyId: K8SSAND-631
┆priority: Medium
